geometry_msgs/PoseStamped goal
bool new_cloud
---
bool success
